test/ginkgo/testsuite_test.go

```diff
@@ -0,0 +1,101 @@
+// Before you run integration tests, you need to set
+// PULL_SECRET_LOCATION environment variable to point to your
+// pull-secret file. In case you are running a non-release binary
+// (with standalone bundle), you also need to set BUNDLE_LOCATION
+// variable to point to the location of the bundle you are using.
+
+package test_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"os"
+	"os/user"
+	"path/filepath"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+)
+
+type VersionAnswer struct {
+	Version          string `json:"version"`
+	Commit           string `json:"commit"`
+	OpenshiftVersion string `json:"openshiftVersion"`
+	Embedded         bool   `json:"embedded"`
+}
+
+type StatusAnswer struct {
+	CRCStatus        string `json:"crcStatus"`
+	OpenshiftStatus  string `json:"openshiftStatus"`
+	OpenshiftVersion string `json:"openshiftVersion"`
+	DiskUsage        int    `json:"diskUsage"`
+	DiskSize         int    `json:"diskSize"`
+	CacheUsage       int    `json:"diskSize"`
+	CacheDir         string `json:"cacheDir"`
+}
+
+var credPath string
+var userHome string
+var versionInfo VersionAnswer
+
+var bundleLocation string
+var pullSecretLocation string
+
+func TestTest(t *testing.T) {
+
+	rand.Seed(time.Now().UTC().UnixNano()) // what's this for?
+
+	RegisterFailHandler(Fail)
+
+	junitfile := fmt.Sprintf("junit_%s.xml", time.Now().Format("2006-01-02 3:4:5"))
+	junitReporter := reporters.NewJUnitReporter(filepath.Join("results", junitfile))
+	RunSpecsWithDefaultAndCustomReporters(t, "Test Suite", []Reporter{junitReporter})
+	//RunSpecs(t, "Test Suite")
+}
+
+var _ = BeforeSuite(func() {
+
+	// set userHome
+	usr, err := user.Current()
+	if err != nil {
+		os.Exit(1)
```

Comment:
`Expect(err).NotTo(HaveOccurred())` instead of os.Exit.

Reply:
I did that, but I remember reading that `Expect` should not be used outside of `It`. Maybe I missed a caveat (like `BeforeSuite` being similar construct as `It`?).